### PR TITLE
Update Lesson 3

### PR DESCRIPTION
**Pull request by viettrung9012**
- [x] This PR contains only information intended for public view

To view this PR's branch in binder:
```
https://mybinder.org/v2/gh/viettrung9012/python-codeacademy-sg/lesson-3?urlpath=lab
```

### CODE REVIEW

File: Lesson3.ipynb [hunk 1]
```diff
@@ -29,6 +32,440 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# With loop\n",
+    "for i in range(10):\n",
```
Comment (puikwan) [inline Lesson3.ipynb]: Is it worth explaining that range(n) in Python means 0, 1, 2... n-1 since Python is 0-index based?

File: Lesson3.ipynb [hunk 2]
```diff
@@ -29,6 +32,440 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Function\n",
+    "\n",
+    "### What is a function?\n",
+    "In computer programming, a function (subroutine) is a sequence of program instructions that perform a specific task, packaged as a unit. This unit can then be used in programs wherever that particular task should be performed. (Wikipedia)\n",
+    "\n",
+    "When I attending a Bluejeans meeting, here are what I usually do:\n",
```
Comment (puikwan) [inline Lesson3.ipynb]: When I attend a Bluejeans meeting, here is what I usually do:

File: Lesson3.ipynb [hunk 3]
```diff
@@ -29,6 +32,440 @@
...
+    "> Open Bluejeans\n",
+    "\n",
+    "> Enter the meeting ID - 4482147579 \n",
```
Comment (puikwan) [inline Lesson3.ipynb]: Should not share your Bluejeans meeting ID?

File: Lesson3.ipynb [hunk 4]
```diff
@@ -29,6 +32,440 @@
...
+    "\n",
+    "> Participate in the meeting\n",
+    "\n",
+    "> Leave the meeting at the end\n",
+    "\n",
+    "> Enter the rating\n",
+    "\n",
+    "These steps are usually similar for my other Bluejeans meetings, except the 2nd step I might enter a different meeting ID\n",
```
Comment (puikwan) [inline Lesson3.ipynb]: except for the 2nd step

File: Lesson3.ipynb [hunk 5]
```diff
@@ -29,6 +32,440 @@
...
+    "\n",
+    "In coding we can define a function like this:\n",
+    "```Python\n",
+    "def join_bluejeans_meeting(meeting_id):\n",
+    "    ...                               # omitted to save space\n",
+    "    enter_meeting_id using meeting_id # making use of the meeting_id in parameter\n",
+    "    ...                               # omitted to save space\n",
+    "    \n",
+    "# if I want to attend meeting 4482147579, I can do\n",
```
Comment (puikwan) [inline Lesson3.ipynb]: should not share your Bluejeans meeting ID?

File: Lesson3.ipynb [hunk 6]
```diff
@@ -29,6 +32,440 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Function to return sum of 3 numbers\n",
+    "def sum_of_three(num1, num2, num3): # we define the function here\n",
```
Comment (puikwan) [inline Lesson3.ipynb:326]: Do we need to explain coding convention? When to use _ and when to use camelBack?

File: Lesson3.ipynb (hunk 5 above)
Comment (viettrung9012) [reply Lesson3.ipynb]: It's yours :P

File: Lesson3.ipynb (hunk 6 above)
Comment (viettrung9012) [reply Lesson3.ipynb:326]: I think it is fine, python convention is _ . If we want convention probably should explain way earlier

**Pull request merged**